services/web/server/src/simcore_service_webserver/projects/_groups_api.py

```diff
@@ -139,13 +139,15 @@ async def delete_project_group(
     group_id: GroupID,
     product_name: ProductName,
 ) -> None:
-    await check_user_project_permission(
-        app,
-        project_id=project_id,
-        user_id=user_id,
-        product_name=product_name,
-        permission="delete",
-    )
+    user = await users_api.get_user(app, user_id=user_id)
+    if user["primary_gid"] != group_id:
```

Comment:
Q: so here in reality you are deleting **the relation** between a project and a group, and not the latter, right?

Reply:
correct :+1: